internal/backend/circuits/cmp.go

```diff
@@ -55,5 +55,5 @@ func init() {
 		},
 	}
 
-	addNewEntry("cmp", &cmpCircuit{}, good, bad, []ecc.ID{ecc.BW6_761})
+	addNewEntry("cmp", &cmpCircuit{}, good, bad, []ecc.ID{ecc.BN254})
```

Comment:
Is there a reason for changing the curve for this test? Why not ecc.Implemented()?

Reply:
I suspect we missed it in PR #233 --> I like to run `go test -short` and with BW6 hard coded here, it takes 10x more time :)